docs/sphinx/quickstart.rst

```diff
@@ -149,6 +149,21 @@ the distributed aspects of |hpx|.
    It is also possible to build e.g. all quickstart examples using ``make
    examples.quickstart``.
 
+Installing and building |hpx| via vcpkg
+=======================================
+
+You can download and install hpx using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
```

Comment:
```suggestion
You can download and install |hpx| using the `vcpkg <https://github.com/Microsoft/vcpkg>` dependency manager:
```

Reply:
Thanks for your review.
Have done as you suggested.